indico/modules/events/models/persons.py

```diff
@@ -346,6 +346,24 @@ def has_role(self, role, obj):
                    if ((self.user_id is not None and self.user_id == x.user_id) or
                        (self.email is not None and self.email == x.email)))
 
+    @property
+    def has_links(self):
+        """Whether the person is an author, speaker, etc., in the event."""
+        chairpersons = {link.person for link in self.event.person_links}
+        if self in chairpersons:
+            return True
+        if (
+            self.event.has_feature('abstracts') and
```

Comment:
I wouldn't check that tbh... it's very much an edge case and showing someone as unused even though they are linked to a non-deleted abstract (even though it's not visible unless the feature is re-enabled) feels wrong